ui/src/components/VolumeOverviewTab.js

```diff
@@ -195,6 +210,8 @@ const VolumeDetailCard = (props) => {
     <VolumeDetailCardContainer>
       <VolumeInformation>
         <VolumeNameTitle data-cy="volume_detail_card_name">
+          <CircleStatus className="fas fa-circle fa-2x" status={health} />
+          &nbsp;
           {name}
         </VolumeNameTitle>
         <InformationSpan>
```

Comment:
You may want to reflector the layout to avoid the label from wrapping as we discussed last time?